life_expectancy/tests/test_cleaning.py

```diff
@@ -0,0 +1,16 @@
+"""Tests for the cleaning module"""
+import pandas as pd
+
+from life_expectancy.cleaning import main
+from . import OUTPUT_DIR
+
+
+def test_clean_data(pt_life_expectancy_expected):
+    """Run the `main` function and compare the output to the expected output"""
+    main(country_code="PT")
```

Comment:
i don't think you should change the argument of your test

Reply:
Yeah, I clearly miss understood what I should do.